war/src/main/webapp/scripts/hudson-behavior.js

```diff
@@ -2242,14 +2242,16 @@ function getStyle(e, a) {
  */
 // eslint-disable-next-line no-unused-vars
 function ensureVisible(e) {
-  var viewport = YAHOO.util.Dom.getClientRegion();
-  var pos = YAHOO.util.Dom.getRegion(e);
-
-  var Y = viewport.top;
-  var H = viewport.height;
+  const scrollTop = Math.max(
```

Comment:
The current scroll position is read from either `body` or `documentElement`, but the new value is only applied to `documentElement`. AFAIK recent browser versions use `documentElement` for the global scroll, so the part with `body` can be omitted?

Reply:
I copied that code from yahoo. Removed the part with body.